nextflow.config

```
@@ -24,7 +41,6 @@ params {
     max_multiqc_email_size     = '25.MB'
 
     // Boilerplate options
-    outdir                     = null
```

Now I see why in your commands you did not include the `outdir` parameter. This parameter was forced to be required and set by default to `null` since otherwise in some cloud environments (e.g. AWS batch) having it set to a relative path as before e.g. `./results` was causing some issues, read more [here](https://github.com/nf-core/tools/issues/1415).
```suggestion
    outdir                     = null
```

---

That make sense now, thanks for the info. 
Corrected